src/global/tracking/tracking.cc

```diff
@@ -214,6 +215,14 @@ void InitPlugin(JApplication* app) {
   app->Add(new JOmniFactoryGeneratorT<CollectionCollector_factory<edm4eic::Track, true>>(
       "CombinedTracks", {"CentralCKFTracks", "TaggerTrackerTracks"}, {"CombinedTracks"}, app));
 
+  app->Add(new JOmniFactoryGeneratorT<SecondaryVertexFinder_factory>(
+      "AMVSecondaryTrackVertices", {"ReconstructedParticles", "CentralCKFActsTrajectories"},
+      {
+          "AMVPrimaryVertices",
+          "AMVSecondaryTrackVertices",
```

Comment:
Don't prefix with name of algorithm. If anything, suffix. Be consistent and don't call something `PrimaryVertices` but the other thing `SecondaryTrackVertices`. Use Track or not, but consistently.